Add missing error message and technique to resolve

ifly7charlie opened the pull request:
<!--
Thanks for your interest in the project. Bugs filed and PRs submitted are appreciated!

Please make sure that you are familiar with and follow the Code of Conduct for
this project (found in the CODE_OF_CONDUCT.md file).

Also, please make sure you're familiar with and follow the instructions in the
contributing guidelines (found in the CONTRIBUTING.md file).

If you're new to contributing to open source projects, you might find this free
video course helpful: https://kcd.im/pull-request

Please fill out the information below to expedite the review and (hopefully)
merge of your pull request!
-->

<!-- What changes are being made? (What feature/bug is being fixed here?) -->

**What**: Documentation Improvement

<!-- Why are these changes necessary? -->

**Why**: It's hard to find the solution to this error, but it is easy to resolve

<!-- How were these changes implemented? -->

**How**: Additional documentation including an example of solution

<!-- Have you done all of these things?  -->

**Checklist**:

<!-- add "N/A" to the end of each line that's irrelevant to your changes -->
<!-- to check an item, place an "x" in the box like so: "- [x] Documentation" -->

- [ x ] Documentation
- [ ] Tests
- [ x ] Ready to be merged
      <!-- In your opinion, is this ready to be merged as soon as it's reviewed? -->

<!-- feel free to add additional comments -->

It would be good to document this in the configuration page as well

ifly7charlie committed:
Add missing error message and technique to resolve

balazsorban44 commented on www/docs/errors.md:
```diff
@@ -77,6 +77,18 @@ The provider you tried to use failed when setting [PKCE or Proof Key for Code Ex
 The `code_verifier` is saved in a cookie called (by default) `__Secure-next-auth.pkce.code_verifier` which expires after 15 minutes.
 Check if `cookies.pkceCodeVerifier` is configured correctly. The default `code_challenge_method` is `"S256"`. This is currently not configurable to `"plain"`, as it is not recommended, and in most cases it is only supported for backward compatibility.
 
+#### ERR_JOSE_ALG_NOT_WHITELISTED
```
I cannot seem to find a reference to this error code in our source code. Are you sure it is originated from `next-auth`?

ifly7charlie replied:
ok, that's embarrassing - correct fix, but I was generating custom JWT in the same method and got the error messages crossed. Actual one is ERR_JWK_KEY_SUPPORT and https://next-auth.js.org/errors#jwt_session_error 

[next-auth][error][jwt_session_error] 
https://next-auth.js.org/errors#jwt_session_error JWKKeySupport: the key does not support HS512 verify algorithm
    at check (/Users/melissa/Documents/GitHub/busyness/busyness/node_modules/jose/lib/jwa/index.js:45:11)
    at check (/Users/melissa/Documents/GitHub/busyness/busyness/node_modules/jose/lib/jwa/index.js:58:12)
    at jwsVerify (/Users/melissa/Documents/GitHub/busyness/busyness/node_modules/jose/lib/jws/verify.js:150:5)
    at Object.module.exports [as verify] (/Users/melissa/Documents/GitHub/busyness/busyness/node_modules/jose/lib/jwt/verify.js:235:5)
    at Object.<anonymous> (/Users/melissa/Documents/GitHub/busyness/busyness/node_modules/next-auth/dist/lib/jwt.js:105:30)
    at Generator.next (<anonymous>)
    at asyncGeneratorStep (/Users/melissa/Documents/GitHub/busyness/busyness/node_modules/next-auth/dist/lib/jwt.js:22:103)
    at _next (/Users/melissa/Documents/GitHub/busyness/busyness/node_modules/next-auth/dist/lib/jwt.js:24:194)
    at /Users/melissa/Documents/GitHub/busyness/busyness/node_modules/next-auth/dist/lib/jwt.js:24:364
    at new Promise (<anonymous>) {
  code: 'ERR_JWK_KEY_SUPPORT'
}

balazsorban44 replied:
so since the error is originated from jwt_session_error, it should be put under there, I think

balazsorban44 reviewed (changes requested):
So I reviewed this, and tried looking for `"ERR_JOSE_ALG_NOT_WHITELISTED"` in our source code, but I don't think we actually use this. Do you maybe have a reproduction where the next-auth log actually sends you to: https://next-auth.js.org/errors#err_jose_alg_not_whitelisted?

Otherwise, this error documentation won't be seen.

ifly7charlie committed:
Update errors.md

Correct with correct error message and more complete suggestions on resolving it
